packages/flutter/lib/src/widgets/form.dart

```diff
@@ -278,8 +282,9 @@ class FormFieldState<T> extends State<FormField<T>> {
   bool get hasError => _errorText != null;
 
   /// Calls the [FormField]'s onSaved method with the current value.
+  /// If the [FormField]'s enabled is false, the method does nothing.
```

Comment:
please put a blank line (prefixed with `///`) between the first sentence (which is used as the summary in the docs) and the second (which will be a second paragraph for when you look at it directly).

Reply:
Blank line added